src/Symfony/Component/Mailer/Transport/Smtp/SmtpTransport.php

```diff
@@ -29,7 +29,7 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Chris Corbyn
  */
-class SmtpTransport extends AbstractTransport
+final class SmtpTransport extends AbstractTransport
```

Comment:
This class has to remain `@final` because we actually extend it. See also the error Psalm produced because of that.

Reply:
not `@final` then if we extend it

Reply:
> not `@final` then if we extend it

Right. I totally missed that the class itself wasn't actually flagged as `@final`, just one protected method.